src/hf_hydrodata/point.py

```diff
@@ -851,6 +854,29 @@ def get_site_variables(*args, **kwargs):
     ]
 
     final_df = final_df[ordered_cols]
+
+    # Merge on conus grid mappings
+    final_site_list = list(final_df["site_id"].unique())
+    conus_map_df = pd.DataFrame(
+        columns=(["site_id", "conus1_i", "conus1_j", "conus2_i", "conus2_j"])
+    )
+
+    for tbl in SITE_ATTRIBUTE_TABLES:
+        conus_map_query = f"""SELECT site_id, conus1_i, conus1_j, conus2_i, conus2_j
+                              FROM {tbl}
+                              WHERE site_id IN (%s)""" % ",".join(
+            "?" * len(final_site_list)
+        )
+        tbl_df = pd.read_sql_query(conus_map_query, conn, params=final_site_list)
+        if len(tbl_df) > 0:
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore", category=FutureWarning)
+                conus_map_df = pd.concat(
+                    [conus_map_df, tbl_df], axis=0, ignore_index=True
+                )
```

Comment:
Just out of curiosity, what does this do? What are `FutureWarnings`?

Reply:
FutureWarnings are notifications for things that are going to change in the next major release (in this case version 3.0.0 of pandas). I am okay with the fore-warned behavior, so this just keeps those repetitive warnings from propagating through to the output console.

Reply:
Oh cool. Thanks for the explanation!